app/Actions/AlbumAuthorisationProvider.php

```diff
@@ -348,7 +356,7 @@ public function applyBrowsabilityFilter(AlbumBuilder $query, ?Album $origin = nu
 
 		// Ensures that only those albums of the original query are
 		// returned for which a path from the origin to the album exist ...
-		if ($origin) {
+		if ($origin != null) {
```

Comment:
```suggestion
		if ($origin !== null) {
```

I will only comment this for a second and last time, see my first comment on this above. We should first discuss, why we only use the weak comparison operator here and if we have come to an answer then change it everywhere in one go if necessary.